app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SessionsController < ApplicationController
+  def index
+    # @symphony_ok = symphony_client.ping
+
+    redirect_to summaries_url if current_user? || request.env['warden'].authenticate(:library_id)
+    # TODO: redirect_to errors_url, alert: 'Unable to authenticate.'
+    # TODO: flash.now.alert = request.env['warden'].message
```

Comment:
What is there todo for this one?

Reply:
for the case if login was not successful, display the message kinda thing 🤔

Reply:
So we can delete, because I created #20 ?

Reply:
sure, will do